src/sse/sse.js

```diff
@@ -102,7 +102,19 @@ This extension adds support for Server Sent Events to htmx.  See /www/extensions
       var sseEventNames = sseSwapAttr.split(',')
 
       for (var i = 0; i < sseEventNames.length; i++) {
-        const sseEventName = sseEventNames[i].trim()
+        let sseEventName
+        let sseSwapStyle
+        const sseEvent = sseEventNames[i].trim()
+        
+        if (sseEvent.indexOf(':') > 0) {
```

Comment:
I'm worried about retro-compatibility though; if people were using event names including colons for whatever reason, this change would suddenly break their app.
Maybe we should use a separate attribute to add such a feature? If it's still worth it considering the `hx-disinherit` mention above

Reply:
you're right, maybe a double column will do? 

like the pattern for `hx-on`

Reply:
Well it's the same issue unfortunately, we can't know what crazy naming people might have been using all this time.
For `hx-on`, it's not an issue as it was designed with this syntax in the first place, so there's no retro compatibility problem. I know it can be frustrating, but we strive to not let any user behind when adding features!

Maybe add a `sse-swap` attribute to control that separately then?

Reply:
pushed another commit that rollsback the `sse-wap` to default and registered `hx-sse-swap` as new tag, let me know if anything and as i'm writing tests for it.

thanks

Reply:
Call me an idiot, I just realized we had a `sse-swap` attribute already 😅 that naming sure would be confusing here.
So, as we already have `sse-connect` and `sse-swap` (I had the lib's attribute naming in mind thus the `hx-` prefix, forgot we don't use it all here), meh, maybe `sse-swap-style` would be a okish name? 😕 

It's unfortunate as `hx-swap` is used in the lib to define the swap style, and `hx-trigger` for the triggering events. Turns out we have the opposite here where `sse-swap` currently specifies the events triggering the SSE swap.

Reply:
mmmm... dunno, a `sse-swap-style` will be confusing 'cause it will define the swap strategy as well as the event trigger, at this point i believe that maybe in future versions the best option is a complete refactor, changing the actual `sse-swap` to the more idiomatic `hx-sse-swap` or `hx-sse-trigger`, and since that will be  a breaking change the swap strategy will not be a problem regarding the backwards compatibility

Reply:
We've waited for htmx 2 to introduce breaking changes, and to be honest, it's very unlikely that we'll do another major version anytime soon, if at all.
> 'cause it will define the swap strategy as well as the event trigger

Why would it need to? `sse-swap` already lets you define the event triggers, even mutiple of them if you want (separated by comma).
What about a `sse-swap-style` attribute that follows the same principle? There's no need to make this new attribute handle triggering events, as triggers are already handled by `sse-swap`, unless I misunderstood something?

Reply:
since my main goal was to reduce the `hx-*` tag and have LOB, i believe that the best option at this point is to keep the extension as is and just use the `hx-swap` to define the strategy, and for the rare cases that needs it use `hx-disinherit`.

maybe in future releases ( where there's also other breaking changes ) this can be reintroduced.

Reply:
Understandable, then I'm closing this PR for now.
Thanks for your patience!